src/api/app/models/buildresult.rb

```diff
@@ -85,6 +85,7 @@ def self.summary(project_name)
         state: state
       )
 
+      StatusCount = Struct.new(:code, :count)
       build.summary = []
       result['summary'].elements('statuscount').each do |count|
         build.summary << StatusCount.new(code: count['code'], count: count['count'])
```

Comment:
[Looks](https://ruby-doc.org/core-2.5.0/Struct.html) like this could be written as ``StatusCount.new(count['code'], count['count'])` now:-)